INSTALL.md

```diff
@@ -195,3 +199,72 @@ For example, in a Linux platform, assuming one wishes to execute the `adaboost_d
 ```shell
 ./_cmake_results/intel_intel64_so/adaboost_dense_batch
 ```
+
+## Conda Development Environment Setup
+
+The previous instructions assumed system-wide installs of the necessary dependencies. These can also be installed at a user-level through the `conda` or [mamba](https://github.com/conda-forge/miniforge) ecosystems.
+
+First, create a conda environment for building oneDAL, after `conda` has been installed:
+
+```shell
+conda create -y -n onedal_env
+conda activate onedal_env
+```
+
+Then, install the necessary dependencies from the appropriate channels with `conda`:
+
+* Linux:
```

Comment:
```suggestion
**Linux\***:
```

Reply:
That doesn't appear to render correctly:
![image](https://github.com/user-attachments/assets/454b38ca-c8ae-4128-aafb-bc2188e322bb)

Reply:
Modified the 4 occurrences to use the formatting you're suggesting but inside bullet points.